app/assets/stylesheets/content/_grid.sass

```diff
@@ -266,3 +266,15 @@ body.widget-grid-layout
     position: sticky
     bottom: 0
     margin-bottom: 0
+
+@include breakpoint(680px down)
+  .grid--container
+    grid-template-columns: auto !important
+    grid-template-rows: auto !important
+    grid-gap: 15px
+    padding: 15px 2px
+
+  .grid--area
+    &.-widgeted
+      grid-column: 1/3 !important
```

Comment:
Why 1/3 and not 1/2?

Reply:
I think it makes sense to move this to a separate file, something like `_grid_mobile.sass`. Thus the file doesn't get too big.